modules/distillery.cpp

```diff
@@ -550,7 +550,9 @@ void LapH::distillery::add_to_perambulator(const size_t dil_t, const size_t dil_
     // checking if smeared or stochastic sink must be computed
     if (!param.dilution_type_si[nbs][1].compare("F")){ // smeared sink
       // running over sink time index
+#pragma omp for schedule(dynamic) nowait
```

Comment:
note that the `nowait` clause here is essential as otherwise the master thread would have to enter here as well because of the implicit barrier at the end of `omp for`